casper/src/main/rholang/MakeMint.rho

```diff
@@ -47,18 +49,27 @@ in {
                 }
               }
             } |
-            contract thisPurse(@"deposit", @amount, @src, success) = {
-              new result in {
-                @src!(*decr, amount, *result) | //call src decr function.
-                for(@decrSuccess <- result) {
-                  if (decrSuccess) {
-                    balance!("add", amount, *success) // add transferred amount to this purse
+
+            contract thisPurse(@"deposit", @amount, @src, successCh) = {
```

Comment:
Is changing `success` to `successCh` worthwhile? Is there a style guide we're trying to follow? It made the diff hard for me to read.

Reply:
I'm not sure if we have a style guide but we should. I wanted to make it clear that the parameter is the return channel